### PR TITLE
Made changes based off issue #929

### LINKED ISSUE #929: Anaconda Tutorial Page Fixes
### What is inaccurate?

1. We will have to include the installation step for the pip package within the conda environment in this section:  https://docs.rc.uab.edu/cheaha/tutorial/pytorch_tensorflow/#install-tensorflow-gpu-using-the-terminal. It would be helpful to reference why this is necessary.
2.  For the PyTorch example, we can add the installation step of ipykernel.
3. Highlighting the `conda env` kernel in the PyTorch example will be useful.
4. The `conda activate` step has to be added at a few places.

Changes to 2 and 3 are for the section: https://docs.rc.uab.edu/cheaha/tutorial/pytorch_tensorflow/#using-pytorch-on-jupyter-notebook.

### Where is the inaccuracy?

https://docs.rc.uab.edu/cheaha/tutorial/pytorch_tensorflow/#anaconda-environment-tutorial-for-pytorch-and-tensorflow

### PR DESCRIPTION
**Pull request by iam4tune**
# Pull Request

For the first point referenced in this issue, we already have this info in our docs. So I created a link to that part of our docs. The pytorch example already has instructions for installing ipykernel in the Note section. I have also referenced a part in our docs, that shows how to switch kernels.

Fixes #929

### CODE REVIEW

File: docs/cheaha/tutorial/pytorch_tensorflow.md [hunk 1]
```diff
@@ -38,24 +38,12 @@ The instructions below, provide a recommended step by step guide to creating and
...
-
-<!-- markdownlint-disable MD046 -->
-!!! note
-
-    When installing packages, modules and libraries into environments, remember to also install `ipykernel` using `conda install ipykernel`. This way your activated environment would appear in the list of kernels in your Jupyter Notebook.
-
-<!-- markdownlint-enable MD046 -->
-
-For a correct installation of pytorch, we have to ensure some conditions are met. See partition [docs](../hardware.md#details) for a guide. One of such conditions, is to load CUDA toolkit using the below command in your environment setup form (see image below).
+1. For a correct installation of pytorch using GPUs, we have to ensure some conditions are met. See partition [docs](../hardware.md#details) for a guide. One of such conditions, is to load the CUDA toolkit using the below command in your environment setup form (see image below).
```
Comment (wwarriner) [inline docs/cheaha/tutorial/pytorch_tensorflow.md:43]: Please clarify "partition docs".

If we tell folks there are "some conditions" then we need to list them all. In a step-by-step instruction, that phrasing "some conditions" could be replaced by explicitly stating each step as they come up in the flow of steps.

I realize this was in the original, but no time like today to fix up ambiguous statements.

File: docs/cheaha/tutorial/pytorch_tensorflow.md [hunk 2]
```diff
@@ -68,18 +56,27 @@ module load CUDA/11.8.0
...
+1. [Create](../../workflow_solutions/using_anaconda.md#create-an-environment) and [activate](../../workflow_solutions/using_anaconda.md#activate-an-environment) an environment as stated in these links.
+
+<!-- markdownlint-disable MD046 -->
+!!! note
+
+    When installing packages, modules and libraries into environments, remember to also install `ipykernel` using `conda install ipykernel`. This way your activated environment would appear in the list of kernels in your Jupyter Notebook. We have a [section](../../cheaha/open_ondemand/ood_jupyter.md#working-with-anaconda-environments) in our docs detailing how to switch conda environments.
+
+<!-- markdownlint-enable MD046 -->
+
+1. When your job has been created and your environment created and activated from the terminal (see above [instructions](../../workflow_solutions/using_anaconda.md#create-an-environment)), run the below command.
```
Comment (wwarriner) [inline docs/cheaha/tutorial/pytorch_tensorflow.md:70]: Let's write "job has started" to set it apart from the "environment created...".

File: docs/cheaha/tutorial/pytorch_tensorflow.md [hunk 3]
```diff
@@ -68,18 +56,27 @@ module load CUDA/11.8.0
...
 
-This commands will install a GPU compatible PyTorch version into your environment. To verify PyTorch is installed, and to see what version you have installed in your environment, use the below command.
+These commands will install a GPU compatible PyTorch version into your environment. To verify PyTorch is installed, and to see what version you have installed in your environment, use the below command.
 
 ```bash
 conda list | grep "torch"
-
 ```
 
 You should get an output like the below image.
```
Comment (wwarriner) [inline docs/cheaha/tutorial/pytorch_tensorflow.md:82]: Write out the contents of the image into a code block for people using screen readers.

File: docs/cheaha/tutorial/pytorch_tensorflow.md [hunk 4]
```diff
@@ -108,25 +104,25 @@ print(torch.cuda.get_device_name(x))
...
 
-1. The TensorFlow CPU and GPU versions requires pip to be up-to-date, to install and upgrade pip to the latest version use the below command.
+1. Activate the conda environment created in Step 1.
 
     ```bash
+    conda activate tensorflow
+    ```
 
-    pip install --upgrade pip
+1. The TensorFlow CPU and GPU versions requires pip to be up-to-date, to install and upgrade pip to the latest version, within your conda environment use the below command. See our [section](../../cheaha/open_ondemand/ood_jupyter.md#pip-installs-packages-outside-of-environment), on why installing pip within a conda environment is advised.
```
Comment (wwarriner) [inline docs/cheaha/tutorial/pytorch_tensorflow.md:116]: I think the first sentence could be two sentences. The first comma would be a good stopping point.

File: docs/cheaha/tutorial/images/pytorch_output.png
Comment (wwarriner) [inline docs/cheaha/tutorial/images/pytorch_output.png:1]: What changed in the image? I can't see a difference in the image, and the "Onion Skin" shows no changes. If we can avoid git thinking the image changed, the repo will be smaller.

**Review by wwarriner — CHANGES_REQUESTED** [review]
This is great! I think while we're in here let's fix up a few ambiguities in language. You've done a great job clarifying what you did change, so let's take advantage of that.

The image shows as changed, but I can't see a difference. Let's figure out how to remove that change from the git history. Or, if there is a change, please let me know what it is.

File: docs/cheaha/tutorial/images/pytorch_output.png
Comment (iam4tune) [reply docs/cheaha/tutorial/images/pytorch_output.png:1]: I don't remember making changes to the image. However, it's possible that my opening and resaving it on my local computer may have altered the metadata, causing Git to detect a change.

**Review by wwarriner — CHANGES_REQUESTED** [review]
Looks great! We're very close on this one. Since we are concluding there are no changes to the image content, only metadata, let's remove the change from the PR. To do this, try what is written here: https://stackoverflow.com/questions/39459467/remove-a-modified-file-from-pull-request
